project/depend.scala

```diff
@@ -1,13 +1,14 @@
 import sbt._
 import Keys._
+import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
 
 object depend {
 
   val catsVersion     = "0.4.0"
   val specs2Version   = "3.6.5"
 
   val cats = Seq(
-      "org.typelevel" %% "cats"
+      "org.typelevel" %%%! "cats"
```

Comment:
plain `%%%` doesnt work, its not well documented but some context at https://github.com/vmunier/play-with-scalajs-example/issues/20